RickAndMorty-iOS/Views/Characters/CharacterDetailsView.swift

```diff
@@ -0,0 +1,155 @@
+}
+
+// MARK: - CollectionView Layout
+extension CharacterDetailsView {
+    private func createLayout() -> UICollectionViewLayout {
+        let layout = UICollectionViewCompositionalLayout { (sectionIndex, _) -> NSCollectionLayoutSection? in
+
+            guard let sectionType = Section(rawValue: sectionIndex) else {
+                return nil
+            }
+
+            let columns = sectionType.columnCount
+
+            let itemSize = NSCollectionLayoutSize(widthDimension: .fractionalWidth(1.0),
+                                                  heightDimension: .fractionalHeight(1.0))
+            let item = NSCollectionLayoutItem(layoutSize: itemSize)
+            item.contentInsets = NSDirectionalEdgeInsets(top: 5, leading: 5, bottom: 5, trailing: 5)
+
+            func getGroupHeight() -> NSCollectionLayoutDimension {
+                switch sectionType {
+                case .appearance, .empty:
+                    return NSCollectionLayoutDimension.estimated(280)
+                case .info, .location, .emptyInfo, .emptyLocation:
+                    return NSCollectionLayoutDimension.estimated(60)
+                default:
+                    return NSCollectionLayoutDimension.estimated(100)
+                }
+            }
+
+            let groupSize = NSCollectionLayoutSize(widthDimension: .fractionalWidth(1.0),
+                                                   heightDimension: getGroupHeight())
+            let group = NSCollectionLayoutGroup.vertical(layoutSize: groupSize, repeatingSubitem: item, count: columns)
+
+            let section = NSCollectionLayoutSection(group: group)
+
+            let headerSize = NSCollectionLayoutSize(widthDimension: .fractionalWidth(1.0), heightDimension: .estimated(50))
+            let header = NSCollectionLayoutBoundarySupplementaryItem(layoutSize: headerSize, elementKind: UICollectionView.elementKindSectionHeader, alignment: .top)
+            if self.collectionView.numberOfItems(inSection: sectionIndex) > 0 {
+                section.boundarySupplementaryItems = [header]
+            }
+            return section
+        }
+        return layout
+    }
+}
+
+extension CharacterDetailsView {
+    func titleView(image: String?, title: String!) -> UIView {
+        let titleWithImage = UIView()
+
+        if let imageUrl = image {
+            let imageView = UIImageView()
+            imageView.sd_setImage(with: URL(string: imageUrl))
+            imageView.layer.cornerRadius = 12
+            imageView.clipsToBounds = true
+            titleWithImage.addSubview(imageView)
+            imageView.snp.makeConstraints { make in
+                make.top.equalToSuperview().offset(-10)
+                make.bottom.leading.equalToSuperview()
+                make.width.equalTo(24)
+                make.height.equalTo(24)
+            }
+        }
+
+        let titleLabel = UILabel()
+        titleLabel.text = title
+        titleLabel.font = UIFont(name: "Creepster-Regular", size: 27)
```

Comment:
You can define couple fonts in extension for Font class and use them, instead of creating each time.